index.rst

```diff
@@ -1,61 +1,150 @@
+   Sketch of a campaign definition screen.
+    
+After defining the campaign, the DRLCM can save it or begin execution by running the first BPS command to submit the first workflow.
+(Initially, command execution will be manual, but eventually it may be integrated into the Campaign Management system.)
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+As the campaign executes, each workflow submission can be assigned one or more Operators to ensure that it is running properly.
+Operators may take shifts, defined by a schedule.
+(Initially, Operator assignments and changes of assignment will be manually entered into the Campaign Management system, but eventually the system may allow a schedule to be defined and automatically applied.)
 
-.. sectnum::
+The progress of each workflow submission is monitored through the workflow system.
+Results of each workflow submission are placed in a Butler Repository (for data products), a metric system (like SQuaSH), or a quality assurance analysis system (for things like plots).
+The Campaign Management system provides links to each of those external systems for easy access.
 
-.. TODO: Delete the note below before merging new content to the master branch.
+.. figure:: /_static/ViewCampaign.png
+    :name: fig-view-campaign
+    :alt: Sketch of a campaign viewing screen.
 
-.. note::
+    Sketch of a campaign viewing screen.
 
-   **This technote is not yet published.**
+A list of all the campaigns can be generated, with sorting and filtering by campaign attributes.
+(Eventually saved queries and the ability to rerun previous campaigns with modifications may be provided.)
 
-   Use cases, scope definition, and proposed implementation for a campaign management system.
+.. figure:: /_static/ListCampaigns.png
+    :name: fig-list-campaigns
+    :alt: Sketch of a campaign list screen.
+
+    Sketch of a campaign list screen.
+
+As each workflow submission completes, the campaign is marked as "Paused" in the Campaign Management system while a determination is made to continue.
+(Initially, this marking will be manual, but eventually it may be integrated into the system.)
+When the entire campaign is complete, it is marked as "Completed" or "Failed" appropriately.
+
+Mini-Data Releases
+------------------
+
+The final configuration for a Data Release will be arrived at via an iterative process.
+Small "development" runs will test out new Science Pipelines features.
+Larger "integration" runs will ensure that unusual data does not cause problems.
+Finally, a "mini-DRP" run, on the scale of 8% of the sky, will be executed to prove the data selection, configuration, and (human) processes.
+The Science Pipelines, System Performance, and Execution teams will meet throughout this process, under the leadership of the DRLCM.
+
+The Campaign Management system will be used for each of these smaller runs, allowing progress (and setbacks) to be monitored, reproduced, and tracked.
```

Comment:
This may just be out of scope for now, but I think a mature version of this document (or a successor to it) needs to attach a schedule to this, working back from the target release date.

I bet even a very rough schedule would focus some helpful attention on the fact (at least that's what I'd call it) that we'll need to be working on multiple data releases at any one time, preparing contingencies for steps that take longer than expected or need to be repeated. and figuring out how to balance near-term and long-term priorities and team structure.

A related question that may be more relevant for even this version of the document: should we be talking about having _two_ campaign management leads, one for DR-next and one for DR-after-that?